tests/pexpects/cursor_selection.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+from pexpect_helper import SpawnedProc
+
+home, end = "\x01", "\x05"
+left, right ="\x02", "\x06"
+select, dump = "\x00", "!"
+
+sp = SpawnedProc()
+sendline, expect_prompt, expect_str = sp.sendline, sp.expect_prompt, sp.expect_str
+expect_prompt()
+
+sendline("bind -k nul begin-selection")
+expect_prompt()
+sendline("bind ! 'echo -n \"<$(commandline --current-selection)>\"'")
+expect_prompt()
+
+sendline("set fish_cursor_selection_mode inclusive")
+
+sendline(f"echo{home}{select}{dump}")
```

Comment:
The test suite is python 3.5+, so no f-strings.

Reply:
> The test suite is python 3.5+, so no f-strings.

Fixed in [6569a0e](https://github.com/fish-shell/fish-shell/pull/8994/commits/6569a0ebd397e8a9f2e1db91e408b512ac4d555e).